ci/bootstrap.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: infrastructure
+value: 1000000
+globalDefault: false
+description: "For infrastructure like gateway."
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: production
+value: 900000
+globalDefault: false
+description: "For production services like apiserver, batch, etc."
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: user
+value: 500000
+globalDefault: true
+description: "For user activities like user batch jobs."
```

Comment:
User jobs are run outside k8s. Is this for pods in PR and dev namespaces?

Reply:
Again, this is old cold pulled from vdc/.  I'll update the comment.  It is for everything that isn't specifically infrastructure or production: that includes tests, notebooks, etc.